Clinical-Genomics/cg
# Update/set ticket and name on microbial order via cli

This PR fixes a problem setting ticket number and name on microbial orders in status-db.

Contact: @annagellerbring 

How to test set ticket:
- [x] install on stage (add instructions on how to as needed)
- [x] us
- [x] `cg set microbial-order [LIMSID] [SIGNATURE] -t/--ticket [TICKETNR]`

Expected outcome:
- [x] as below
```
Ticket for [LIMSID] set to [TICKETNR].
Comment added to order [LIMSID]
```
![image](https://user-images.githubusercontent.com/6697636/68574203-ab493400-0469-11ea-8561-96a536d77914.png)

- [x] microbial order ticket has been set to the specified nr, 
![image](https://user-images.githubusercontent.com/6697636/68574277-cd42b680-0469-11ea-80b1-808c2723d2da.png)
->
![image](https://user-images.githubusercontent.com/6697636/68574323-e186b380-0469-11ea-8d2a-a8b44e67ff3d.png)


- [x] an comment has been added to the order specifying who 
- [x] did what and 
- [x] when

How to test set name:
- [x] us
- [x] `cg set microbial-order [LIMSID] [SIGNATURE] -n/--name [NAME]`

Expected outcome:
- [x] as below
```
Name for [LIMSID] set to [NAME].
Comment added to order [LIMSID]
```
![image](https://user-images.githubusercontent.com/6697636/68576023-91a9eb80-046d-11ea-8d2a-0e40192ffdfc.png)

- [x] microbial order name has been set to the specified name, 
![image](https://user-images.githubusercontent.com/6697636/68574518-5eb22880-046a-11ea-800d-83e64365fb0c.png)
->
![image](https://user-images.githubusercontent.com/6697636/68576060-a71f1580-046d-11ea-85ae-d3e941898ef1.png)

- [x] an comment has been added to the order 
- [x] specifying who did 
- [x] what and 
- [x] when
- [x] Project name has been set to [NAME] in LIMS for project with [LIMSID] 
![image](https://user-images.githubusercontent.com/6697636/68576147-d6358700-046d-11ea-86fd-3eb3d4bd64eb.png)


This is a minor **version bump** because it expands the interface in a backwards compatible manner
